README.md

```diff
@@ -62,4 +62,53 @@ This will start a local development server on port 5000 (http://localhost:5000).
 This project welcomes contributors. If you are interested, please feel free to
 join [the mailing list](https://mail.mozilla.org/listinfo/bugsahoy-devel)
 
+## Add a project
+
+Each of the projects is a file in the [data](https://github.com/mozilla-frontend-infra/codetribute/tree/master/src/data)
+folder - to add a new one, create a new file named after the project, ending in `.yaml`. Ensure all spaces and special
+characters are replaced with `-`. Make sure too that the project hasn't already been inside the [data](https://github.com/mozilla-frontend-infra/codetribute/tree/master/src/data)
+folder
+
+The contents of the file are just some details about the project:
+
+```yaml
+name: [project name]
+summary: [a brief description of the project]
+[ The introduction contains more information about the projects in markdown format.
+Below are the suggestions on how to write the introduction. ]
+introduction: |
+  ## About [project name]
+
+  ...
+
+  ## Who Works on [project name]?
+
+  ...
+
+  ## How Do I Get Started?
+
+  ...
+
+  ### How Do I Write the Code?
+
+  ...
+
+  ## How Do I Get Help?
+
+  ...
+
+products:
+- [bugzilla product] OR
+- [bugzilla product] : [bugzilla component, specify this if it is not going to be for all component]
+repositories:
+- [repository name] : [issues' tag / label to get the bug]
+[
+    if there is more than one tag for a repository, list them in a different line, e.g
+    - taskcluster/taskcluster-queue: good-first-bug
+    - taskcluster/taskcluster-queue: easy-fix
+]
+```
+
+Check out the [Taskcluster](https://github.com/mozilla-frontend-infra/codetribute/blob/master/src/data/taskcluster.yaml)
+file for an example of this project structure.
```

Comment:
We are going to refer to this file before showing the template file. We can remove this.